recipes/tobac/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "tobac" %}
+{% set version = "1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/climate-processes/{{name}}/archive/v{{version}}.tar.gz
+  sha256: 74a47a181e9a4b4df7f993165ac31aeb870efcdd8c3ffe6b1da4c963f80eb997
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - numpy
+    - scipy
+    - scikit-image
+    - pandas
+    - pytables
+    - matplotlib 
+    - iris 
+    - xarray 
+    - cf_units
+    - cartopy
+    - trackpy
+
+test:
+  imports:
+    - tobac
+    - tobac.testing
+    - numpy
+    - pandas
```

Comment:
it is not necessary to import ``pandas`` and ``numpy``

Reply:
I have removed it from the test section.